Formula/pypy3.7.rb

```diff
@@ -68,7 +53,7 @@ def install
     python = Formula["pypy"].opt_bin/"pypy"
     cd "pypy/goal" do
       system python, buildpath/"rpython/bin/rpython",
-             "-Ojit", "--shared", "--cc", ENV.cc, "--verbose",
+             "-Ojit", "--shared", "--cc", ENV.cc,
              "--make-jobs", ENV.make_jobs, "targetpypystandalone.py"
 
```

Comment:
If you are running the package script on lines 65-69, you do not need to rebuild the dependencies on lines 59-61, it is done inside the package script.

Also, to work around the tkinter problems, you can avoid packaging it by setting `PYPY_PACKAGE_WITHOUTTK=1`. This will cause the rebuilding to skip the _tkinter module